core/lib/spree/core/dependencies.rb

```diff
@@ -21,7 +21,7 @@ class Dependencies
         :order_approve_service, :order_cancel_service, :shipment_change_state_service, :shipment_update_service,
         :shipment_create_service, :shipment_add_item_service, :shipment_remove_item_service,
         :payment_create_service, :address_create_service, :address_update_service,
-        :checkout_select_shipping_method_service
+        :checkout_select_shipping_method_service, :export
```

Comment:
this name can be to general, maybe we could rename that i.e `export_rss` || `export_produts_to_rss`, wdyt ?

Reply:
That would be actually a good idea. But I'm thinking to maybe put it in folder like export_rss and rename the service itself to something like google_rss? Becouse it is specific to google marketplace requirerments and naming conventions and won't be compatible with something like Facebook

Reply:
you thinking about create SO in follow namespace: `ExportRss::Google` ?

Reply:
Yes, something like that

Reply:
cool, let's do that 🤙🏻